examples/specific/struct_function.h

```diff
@@ -10,6 +10,8 @@ struct MyClass
 
     //! \brief struct copy constructor
     MyClass(const MyClass&);
+
+	int myMemberVar;
```

Comment:
It appears the rest of this specific file uses 4 spaces to indent instead of a tab.

Reply:
Good catch, fixed.